examples/hono/package.json

```diff
@@ -5,6 +5,7 @@
   },
   "dependencies": {
     "@upstash/qstash": "latest",
+    "@upstash/workflow": "^0.1.0",
```

Comment:
Shouldnt we get the package from `../../dist` as above?

Reply:
updated all imports with canary